proto/ibc/apps/cross-chain-validation/ccv.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package ibc.apps.ccv.v1;
+
+option go_package = "github.com/cosmos/ibc-go/modules/apps/ccv/types";
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+// This packet is sent from parent chain to baby chain if the validator set for baby chain
+// changes (due to new bonding/unbonding messages or slashing events)
+// The acknowledgement from baby chain will be sent asynchronously once unbonding period is over,
+// and this will function as `UnbondingOver` message for this packet.
+message ValidatorSetChangePacketData {
```

Comment:
This is the only packet that will get sent across the ORDERED channel so sequence number is implicitly enforced by IBC.

We can implement the UnbondingOver message as an asynchronous acknowledgement of this packet. This fits very well with the semantic purpose of the acknowledgement and will make things simpler.

Reply:
I assume there will also be (un/re)delegation requests from follower chain -> leader chain? Should we write the packet data too for them?

Reply:
No I believe all (un/re)delegation messages should be sent to parent chain. The resulting valset changes will be communicated to baby chain through IBC packet

Reply:
would be good to specify in the comments that this is to be used only on an ORDERED channel